stdlib/typing_extensions.pyi

```diff
@@ -155,7 +160,21 @@ def get_type_hints(
     include_extras: bool = False,
 ) -> dict[str, Any]: ...
 def get_args(tp: Any) -> tuple[Any, ...]: ...
-def get_origin(tp: Any) -> Any | None: ...
+
+if sys.version_info >= (3, 10):
+    @overload
+    def get_origin(tp: ParamSpecArgs | ParamSpecKwargs) -> ParamSpec: ...
```

Comment:
Oh, actually, this overload can work on 3.7+ for `typing_extensions` (but _not_ for `typing`, where `ParamSpec`s only exist on 3.10+):

```pycon
(py37) C:\Users\alexw\coding\typeshed>python
Python 3.7.16 (default, Jan 17 2023, 16:06:28) [MSC v.1916 64 bit (AMD64)] :: Anaconda, Inc. on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> from typing_extensions import ParamSpec, get_origin
>>> type(get_origin(ParamSpec("P").args))
<class 'typing_extensions.ParamSpec'>
```

Reply:
Will take care of it later today. Thanks!